mmv1/products/securitycenterv2/OrganizationNotificationConfig.yaml

```diff
@@ -32,6 +32,9 @@ self_link: '{{name}}'
 create_url: 'organizations/{{organization}}/locations/{{location}}/notificationConfigs?configId={{config_id}}'
 update_verb: 'PATCH'
 update_mask: true
+import_format:
+  - '{{project}} {{name}}'
```

Comment:
why does this one need `{{project}}` when the other two similar ones didn't?

Reply:
This one is wrong